epidose/device/update_filter_d.sh

```diff
@@ -64,17 +56,15 @@ get_filter()
   log "Obtaining new filter from $SERVER_URL"
   while : ; do
     wifi_acquire
-    if err=$(curl --silent --show-error --fail --output "$FILTER.new" \
-      "$SERVER_URL/filter" 2>&1) ; then
-      wifi_release
-      # Atomically replace existing filter with new one
-      mv "$FILTER.new" "$FILTER"
-      log "New filter obtained: $(stat -c %s "$FILTER") bytes"
+
+    # Tries to get a new cuckoo filter from the ha-server
+    local exit_code=$(is_trying_to_get_new_filter)
```

Comment:
This is getting the output, not the exit code.

Reply:
So I guess here something like this is needed, right?

```
is_try_to_get_new_filter
local exit_code=$?
```